ci/travis/before_install-linux.sh

```diff
@@ -32,21 +32,18 @@ if [ $CC == 'gcc' ]; then
     export CXX='g++-4.8'
 fi
 
-echo ">>> Installing nupic-linux64..."
-git clone https://github.com/numenta/nupic-linux64.git
```

Comment:
The nupic-linuxXX and nupic-darwinXX environments have gotten hopelessly out of date, and are not needed for the build to succeed.

CC @scottpurdy